server/models/Column.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const Card = require('./Card');
+
+const ColumnSchema = new Schema({
+	name: {
+		type: String,
+		required: true
+	},
+	cards: [Card.schema]
+}, {collection: 'Column'});
+
+// creates a new instance of the Column model and returns the saved instance
+ColumnSchema.statics.createNewColumn = async function(name) {
```

Comment:
Since you have this static, it would be good to use it in the Board schema when you create a column, same with the deleteColumn